src/api/app/views/webui2/webui/staging_workflows/request_ignorations/_create_dialog.html.haml

```diff
@@ -0,0 +1,17 @@
+.modal.fade#ignore-request-modal{ tabindex: -1, role: 'dialog', aria: { labelledby: 'ignore_request-modal-label', hidden: true } }
+  .modal-dialog.modal-dialog-centered{ role: 'document' }
+    .modal-content
+      .modal-header
+        %h5.modal-title#ignore-request-modal-label Ignore Request
+        %a.close{ type: 'button', data: { dismiss: 'modal' }, aria: { label: 'Close' } }
+          %span{ aria: { hidden: true } } &times;
+      = form_for [@staging_workflow, RequestIgnoration.new] do |f|
+        .modal-body
+          .form-group
+            = f.label :bs_request_id, 'Request number'
+            = f.number_field :bs_request_id, required: true, placeholder: 'e.g. 1234', class: 'form-control'
```

Comment:
[nitpick] Does this `e.g. 1234` make sense? :thinking:

Reply:
> [nitpick] Does this `e.g. 1234` make sense? thinking

What do you propose? To remove the placeholder?

Reply:
Yes